### PR TITLE
Upgraded graph-teams-tag nodejs sample to use delagated permissions

### PR DESCRIPTION
**Pull request by Prithvi-MSFT**

### CODE REVIEW

File: samples/graph-teams-tag/nodejs/client/src/components/create-tag.jsx [hunk 1]
```diff
@@ -76,26 +80,62 @@ const CreateTag = props => {
...
-                    microsoftTeams.dialog.submit("Created successfully!");
-                }
-            }
+        var createTagDto;
+        microsoftTeams.app.initialize().then(() => {
+            microsoftTeams.app.getContext().then((context) => {
+                if (tagName !== "" && tagDescription !== "") {
+                            var membersToBeAdded = addSelfIfNotAdded(context.user.id);
+            
+                            createTagDto = {
```
Comment (ChetanSharma-msft) [inline samples/graph-teams-tag/nodejs/client/src/components/create-tag.jsx:89]: Rename this variable "createTagDto" as "createNewTag" at all the places.

**Review by ChetanSharma-msft — CHANGES_REQUESTED** [review]
Added my review comment.

File: samples/graph-teams-tag/nodejs/client/src/components/create-tag.jsx (hunk 1 above)
Comment (Prithvi-MSFT) [reply samples/graph-teams-tag/nodejs/client/src/components/create-tag.jsx]: Changed it

**Review by ChetanSharma-msft — APPROVED** [review]
Changes looks fine, approving it.

**Review by Wajeed-msft — APPROVED** [review]

**Comment by ChetanSharma-msft** [comment]
Executed build pipeline: https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/3604695674
Result:

**Pull request merged**